common4j/src/main/com/microsoft/identity/common/java/providers/nativeauth/responses/resetpassword/ResetPasswordContinueApiResponse.kt

```diff
@@ -0,0 +1,118 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.java.providers.nativeauth.responses.resetpassword
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+import com.microsoft.identity.common.java.logging.LogSession
+import com.microsoft.identity.common.java.providers.nativeauth.IApiResponse
+import com.microsoft.identity.common.java.providers.nativeauth.interactors.InnerError
+import com.microsoft.identity.common.java.util.isExpiredToken
+import com.microsoft.identity.common.java.util.isInvalidOOBValue
+import com.microsoft.identity.common.java.util.isRedirect
+import java.net.HttpURLConnection
+
+/**
+ * Represents the raw response from the Reset Password /continue endpoint.
+ * Can be converted to ResetPasswordChallengeApiResult using the provided toResult() method.
+ */
+class ResetPasswordContinueApiResponse(
+    @Expose override var statusCode: Int,
+    @Expose @SerializedName("password_reset_token") val passwordResetToken: String?,
+    @Expose @SerializedName("password_submit_token") val passwordSubmitToken: String?,
```

Comment:
Do we need both of these tokens? Also shouldn't be exposed

Reply:
I have removed the attribute. I will check if we need both fields.

Reply:
`passwordResetToken` wasn't used. I have removed it.